scripts/apps/monitoring/controllers/AggregateCtrl.ts

```diff
@@ -521,4 +559,20 @@ export function AggregateCtrl($scope, desks, workspaces, preferencesService, sto
     $scope.$on('open:resend', (evt, item) => {
         $scope.resend = item;
     });
+
+    function getActiveProfiles() {
+        content.getTypes(false).then((profiles) => {
+            self.loading = false;
+            self.activeProfiles = profiles;
+
+            // initialize the activeFilterTags once the activeProfiles are available
+            if (self.activeFilters.contentProfile.length > 0) {
+                self.activeFilterTags['Content profile'] = self.activeFilters.contentProfile.map((filter) => {
```

Comment:
here the Content profile is not translated while it is previously, would be good to store it as constant translated and use that